recipes/tensorflow-datasets/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "tensorflow-datasets" %}
+{% set version = "4.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 93a89cd599b24e9cb04a3a3b43cc72728fc11889f41e9d69f0bd95a8318531bc
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - tfds = tensorflow_datasets.scripts.cli.main:launch_cli
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - absl-py
+    - attrs >=18.1.0
+    - dataclasses
+    - dill
+    - future
+    - importlib_resources
+    - numpy
+    - promise
+    - protobuf >=3.12.2
+    - python >=3.6
+    - requests >=2.19.0
+    - six
+    - tensorflow-metadata
```

Can you also add `tensorflow-metadata` to conda-forge? Seems like it isn't packaged yet. CI also suggests that this needs a stricter tensorflow pin. Windows is OK to fail but not with the current error ;)
```suggestion
    - tensorflow-metadata
    - tensforflow >=2.1
```